docs/device_registry_index.md

```diff
@@ -42,7 +42,6 @@ Although not currently available, we could consider offering an option to users
 | model                | The model name of the device.                                                                                                                                                                                                           |
 | model_id             | The model identifier of the device.                                                                                                                                                                                                     |
 | serial_number        | The serial number of the device. Unlike a serial number in the `identifiers` set, this does not need to be unique.                                                                                                                      |
-| suggested_area       | The suggested name for the area where the device is located.                                                                                                                                                                            |
```

Comment:
I think we should document somewhere that it's possible to pass this parameter to `DeviceInfo` and `DeviceRegistry.async_get_or_create`.